ui/eureka/eureka.js

```diff
@@ -650,23 +546,31 @@ let Options = {
         ko: '파고스 정령',
       },
       nms: {
+        northbunny: {
```

Comment:
Can you verify that importing from the tracker (click the red clipboard, past in game with an /echo) works properly? I get an error because northbunny (and other bunnies) don't have a trackerName entry.  I think the right fix is to adjust the function to handle entries with no trackerName (and not to add empty ones here).

Reply:
Good catch, fixed as you suggested.